docs/core/extensions/snippets/logging/console-formatter-custom/CustomFormatter.cs

```diff
@@ -25,17 +25,9 @@ public CustomFormatter(IOptionsMonitor<CustomOptions> options)
             in LogEntry<TState> logEntry,
             IExternalScopeProvider scopeProvider,
             TextWriter textWriter)
-        {
-            if (logEntry.Exception is null)
-            {
-                return;
-            }
-
-            string message =
-                logEntry.Formatter(
-                    logEntry.State, logEntry.Exception);
-            
-            if (message == null)
+        {   
+            string message = logEntry.Formatter(logEntry.State, logEntry.Exception);
+            if (logEntry.Exception == null && message == null)
```

Comment:
I disagree with this proposed change.

If the `logEntry.Exception` is `null`, we should not be calling:

```csharp
logEntry.Formatter(logEntry.State, logEntry.Exception);
```

1. The evaluation is unnecessary as the method will `return`
2. It's added overhead (even though minimal)
3. It knowingly is passing `null` to the `.Formatter` method

Reply:
Looks good! I have committed the suggested changes. Thanks for the review :)